model_compression_toolkit/core/pytorch/graph_substitutions/substitutions/functional_batch_norm.py

```diff
@@ -37,7 +38,7 @@ def __init__(self):
         super().__init__(matcher_instance=bn_node)
 
     @staticmethod
-    def get_attributes_from_weights(node: BaseNode) -> Dict:
+    def get_attributes_from_weights(node: FunctionalNode) -> Dict:
         """
         convert functional batch_norm positional weights to BatchNorm2d weights
```

Comment:
I dont understand why would the BN weights be positional and not regular parameters of the layer (maybe talk about this offline)